universal-application-tool-0.0.1/app/controllers/admin/AdminProgramBlocksController.java

```diff
@@ -123,6 +124,9 @@ public Result move(Request request, long programId, long blockId) {
       programService.moveBlock(programId, blockId, direction);
     } catch (ProgramNotFoundException e) {
       return notFound(e.toString());
+    } catch (IllegalBlockMoveException e) {
```

Comment:
(super nit) can you move this up since its an exception we expect to see often, as opposed to the other ones which should never really happen?